azure-pipelines.yml

```diff
@@ -0,0 +1,189 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+variables:
+  # The $(FlutterToolPath) variable is generated by the FlutterInstall task 
+  # See code in the Flutter Build task https://github.com/aloisdeniel/vsts-flutter-tasks/blob/master/tasks/build/index.ts
+  flutterExecPath: $(FlutterToolPath)/flutter.bat
+  versionNumberRegex: '(?<=version: ).*'
+
+trigger:
+  batch: 'true'
+  branches:
+    include:
+      - develop
+      - release
+      - hotfix
+      - master
+  paths:
+    include:
+      - device_calendar/*
+
+stages:
+  # ----------- CI -----------
+  - stage: Build
+    jobs: 
+     - job: BuildAndroidAndIos
+       pool:
+        vmImage: 'macOS-latest'
+       steps:
+        - task: FlutterInstall@0
+          displayName: 'Flutter install'
+          inputs:
+            channel: 'stable'
+            version: 'latest'
+
+        - task: FlutterBuild@0
+          displayName: 'Flutter build - Android'
+          inputs:
+            target: 'aab'
+            projectDirectory: 'device_calendar/example'
+
+        - task: FlutterBuild@0
+          displayName: 'Flutter build - iOS'
+          inputs:
+            target: 'ios'
+            projectDirectory: 'device_calendar/example'
+            iosCodesign: false
+            
+  # -----------CD Pre-release -----------
+  - stage: DevelopmentRelease
+    dependsOn: Build
+    condition: and(succeeded(), in(variables['Build.SourceBranch'], 'refs/heads/release', 'refs/heads/hotfix'))
+    jobs:
+     - job: ReleaseDevelopmentVersion
+       pool:
+        vmImage: 'windows-latest'
+       steps:
+        - task: FlutterInstall@0
+          displayName: 'Flutter install'
+          inputs:
+            channel: 'stable'
+            version: 'latest'
+
+        - task: PowerShell@2
+          condition: eq(variables['Build.SourceBranch'], 'refs/heads/release')
```

Comment:
Do we need this condition here - surely we want to do this for hotfix as well as release branches?

Reply:
That condition (`condition: eq(variables['Build.SourceBranch'], 'refs/heads/release')`) is only true when the branch is `release` isn't it? Yes we only want to run the task if the branch is `release`, not `hotfix`

Reply:
Updated this to publish dev versions for `hotfix` branch as well

Reply:
As discussed, remove this condition